packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/components/Choices/components/ChoiceButton.js

```diff
@@ -26,10 +26,10 @@ function ChoiceButton({
 
   const choiceButton = useRef(null)
   const handleClick = useCallback(() => {
-    onChoose(choiceId)
+    !disabled && onChoose(choiceId)
   }, [choiceId, onChoose])
   const handleKeyDown = useCallback((event) => {
-    onKeyDown(choiceId, event)
+    !disabled && onKeyDown(choiceId, event)
```

Comment:
Similarly, disabled buttons can’t respond to the keyboard either. They’re removed from the tab order in the DOM.